global.json

```
@@ -0,0 +1,9 @@
+{
+  "tools": {
+    "dotnet": "6.0.100"
```

What requires a .NET SDK be installed w/o enforcing a version❔
```suggestion
  "sdk": {
    "version": "6.0.100"
  },
  "tools": {
    "dotnet": "6.0.100"
```
Might also be worthwhile adding `"allowPrerelease"` and `"rollForward"` values to make it easier to build on random people's machines.